src/pe_reports/report_generator.py

```diff
@@ -11,6 +11,7 @@
   -l --log-level=LEVEL              If specified, then the log level will be set to
                                     the specified value.  Valid values are "debug", "info",
                                     "warning", "error", and "critical". [default: info]
+  -sc --soc_med_included            Include social media posts from Cybersixgill in the report.
```

Comment:
Also, you need to add this optional flag to the ["Usage" line above](https://github.com/cisagov/pe-reports/blob/18e3d1c69fccc85afc0506ed7382e3691b9c5c3d/src/pe_reports/report_generator.py#L4) so that docopt knows about it:
```
pe-reports REPORT_DATE OUTPUT_DIRECTORY [--log-level=LEVEL] [--soc_med_included]
```

Reply:
Issue addressed in the following commit: [ec0f8be](https://github.com/cisagov/pe-reports/pull/565/commits/ec0f8bea0e02715129c7e6e197e01b7e07b00682)